app/services/analytics_service.rb

```diff
@@ -1,14 +1,12 @@
 class AnalyticsService
   class << self
     def call(event_type:, page:, assessment_code:, browser_id:)
-      hash = {
-        key: "CCQ Analytics Datum",
+      AnalyticsEvent.create(
```

Comment:
should this be create! (although if it fails, I'm not sure I want a hard crash...?)

Reply:
Good idea! I'll add a `!` and then catch and Sentrify any exceptions generated.